src/dead_link_checker/core.clj

```diff
@@ -0,0 +1,35 @@
+(ns dead-link-checker.core
+  (:require [clojure.string :as str]
+            [clojure.xml    :as xml]
+            [clojure.zip    :as zip]))
+
+(def test-comment "This is a test comment containing a <a href=\"https://www.google.com\"/> link!")
+
+(defn zip-str [s]
+  (zip/xml-zip 
+   (xml/parse (java.io.ByteArrayInputStream. (.getBytes s)))))
+
+(defn wrap-comment [s] (str "<p>" s "</p>"))
+
+(defn is-html-link [t] (and (map? t) (contains? t :tag) (= (:tag t) :a)))
+
+(defn url-from-html-link [l] (:href (:attrs l)))
+
+(defn links-from-item-content [s] 
```

Comment:
I think this gets convoluted when there's so much nesting. Thoughts on using `->>`?

Reply:
I don't think its too confusing for Clojure in general. I think that is one of the commonalities of Clojure is nesting, and it will take some time for us to get use to that syntax and expressivity. I need to do some research on the `->>` operator.

Reply:
Ah I guess. I'm used to reading top down rather than bottom up (if it spans multiple lines)